### PR TITLE
Improve the docker setup

### PR DESCRIPTION
**Pull request by stof**
Due to layer caching, apt-get update would not run again when changing the list of installed packages if they are in separate layers.

The second commit reads pip requirements from the requirements.txt file instead of duplicating them, making things easier to maintain (no need to change them in 2 places)

### CODE REVIEW

File: support/build/_docker/cedar-14.Dockerfile [hunk 1]
```diff
@@ -3,9 +3,10 @@ FROM heroku/cedar:14
...
-RUN apt-get install -y python-pip
+RUN apt-get update && apt-get install -y python-pip
 
-RUN pip install 'bob-builder>=0.0.10' 's3cmd>=1.6.0'
+COPY requirements.txt /app/requirements.txt
 
-ADD . /app
+RUN pip install -r /app/requirements.txt
+
+COPY . /app
```
Comment (dmathieu) [inline support/build/_docker/cedar-14.Dockerfile]: Why change this from `ADD` to `COPY`?

Comment (dzuelke) [inline support/build/_docker/cedar-14.Dockerfile]: @dmathieu: "For other items (files, directories) that do not require ADD’s tar auto-extraction capability, you should always use COPY.", see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy

Comment (dmathieu) [inline support/build/_docker/cedar-14.Dockerfile]: 👍

**Comment by dmathieu** [comment]
LGTM

**Comment by stof** [comment]
@dzuelke @dmathieu ping

**Comment by stof** [comment]
@dzuelke any news about this PR ?

**Pull request merged**